atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellDeleter.java

```diff
@@ -115,9 +114,8 @@ public String toString() {
                             + numVersions + " total versions of " + cellVersionsMap.size() + " keys)";
                 }
             });
-        } catch (UnavailableException e) {
-            throw new InsufficientConsistencyException("Deleting requires all Cassandra nodes to be up and available.",
-                    e);
+        } catch (RetryLimitReachedException e) {
```

Comment:
is the original ever thrown / do we still want to preserve that behaviour? I think it's not and so it's fine to replace the entire block, but would like a sanity check

Reply:
Calls to runWithRetryOnHost will generally throw RetryLimitReachedException, we won't be able to get Unavailable at this level